### DISCUSSION TITLE
F3 Search results to black font with Daylight.css

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3215] [Q&A]
I have tried to change css-color to rgb(0,0,0), but no effect. How F3 search results (list of answers) font could be black. I use daylight.css and backround of dialog is light and list text is white. How to change it black? Which setting it is?

**Comment by 10110111** [comment]
This worked for me. Not sure of the separately loadable style sheets.

```diff
diff --git a/data/gui/normalStyle.css b/data/gui/normalStyle.css
index 780594159b..ef1bfd7330 100644
--- a/data/gui/normalStyle.css
+++ b/data/gui/normalStyle.css
@@ -917,6 +917,7 @@ QLineEdit#lineEditSearchSkyObject {
 
 QListView#searchListView {
     font-size: 108%;
+    color: rgb(0,0,0);
     padding-left: 20px;
     padding-right: 15px;
     border: none;
```

**Comment by ilkant** [comment]
Thank you! It works.

**Comment by ilkant** [comment]
How about Astronomical Calculators Planetary Calculator texts? They are like white and daylight.css background does not make contrast. So they are hard to read. How to change texts to black? Like this:

QLabel#planetaryCalculator {
        color: rgb(0, 0, 0)
}

**Reply by 10110111** [reply]
I don't know what exactly you mean. You can try using Qt Designer to open `src/gui/astroCalcDialog.ui`, and find the name and type of the widget you need in the widget tree.

**Comment by ilkant** [comment]
I tried to find the color of text with Qt Designer, but didn't find.

I want Astronomical Calculator PC text to be black. So the text would be easy to read when using daylight.css style. Maybe all texts have to be black. But that is not bad thing.

**Reply by 10110111** [reply]
You can replace all labels+checkboxes in AstroCalc dialog (that are not overridden further) like this:
```diff
diff --git a/data/gui/normalStyle.css b/data/gui/normalStyle.css
index 780594159b..f585a3f5de 100644
--- a/data/gui/normalStyle.css
+++ b/data/gui/normalStyle.css
@@ -885,7 +885,7 @@ QLineEdit[collision="true"] {
 
 #astroCalcDialogForm QCheckBox,
 #astroCalcDialogForm QLabel {
-    color: rgb(238, 238, 238);
+    color: rgb(0, 0, 0);
 }
 
 QLabel#monthlyElevationTimeInfo {
```

**Reply by ilkant** [reply]
Thanks! That works.
